dotCMS/src/main/java/com/dotmarketing/portlets/containers/business/ContainerFactoryImpl.java

```diff
@@ -514,7 +514,7 @@ private boolean containsContentType (final FileAssetContainer fileAssetContainer
 	/**
 	 * Finds the container.vtl on a specific host
 	 * returns even working versions but not archived
-	 * the search is based on the ES Index.
+	 * the search is based on the DB.

```

Comment:
The logic modified on this class doesn't contain tests

Reply:
This is a private method so tests explicitly calling this method are not developed. This is being called on all tests that are using/creating file asset containers.